dev/jobs/prep.sh

```diff
@@ -131,21 +146,23 @@ if [[ ${MAKE_PREPBUFR} = "YES" ]]; then
     "${HOMEobsproc}/jobs/JOBSPROC_GLOBAL_PREP" && true
     err=$?
     if [[ ${err} -ne 0 ]]; then
-       echo "FATAL ERROR: Global prep job failed!"
-       exit 1
+       err_chk "FATAL ERROR: Global prep job failed!"
```

Comment:
```suggestion
       err_exit "JOBSPROC_GLOBAL_PREP job failed!"
```
per @DavidHuber-NOAA 's new issue he has opened.

Reply:
Though, it also requires `export err` on line 147

Reply:
Updated @ 7aa4f5982